Crowdspot Docs

samgutentag opened the pull request.

samgutentag committed:
Crowdspot docs initial addition

cokes518 commented:
Does it make more sense to move the docs under "Mine HNT" instead of "Use the Network"?

samgutentag committed:
login flow with sample images

samgutentag commented:
> Does it make more sense to move the docs under "Mine HNT" instead of "Use the Network"?

Works for me! I only needed to give it a home to get started.

samgutentag committed:
relocated page

samgutentag committed:
reorder login flow

samgutentag committed:
evaluation metrics and explainer text

samgutentag committed:
How to Submit to the Denylist

samgutentag committed:
image optimizations

samgutentag committed:
more images

samgutentag committed:
finished docs

- reorder sections to put submissions at head
- included voting instructions and images
- updated evaluation process statuses

samgutentag committed:
formatting fixes and updated Trust Score explainer

samgutentag committed:
reorder sections
